JsonLogic.Tests/GithubTests.cs

```diff
@@ -135,7 +135,7 @@ public void Issue313_VarEqualsNull()
   ]
 }");
 
-		var rule = node.Deserialize<Rule>(LogicSerializerContext.OptionsManager.SerializerOptions);
+		var rule = node.Deserialize(JsonLogicSerializerContext.Default.Rule);
```

Comment:
Not sure if the project structure allows this, but creating separate contexts for each logical grouping of types is bound to create _some_ duplication (e.g. each grouping emits code for common JsonTypeInfo types such as string). If possible, it might make sense to pile as many types as possible in a single context so that common transitive dependencies are shared.

Reply:
Oh, yeah that context is internal, but the test project can see the internals. I need to scan for context usages like that and not do it.  Good catch.